From fc7221d243c44be917dc21cbdc70df7d14e4ce4d Mon Sep 17 00:00:00 2001
From: Jozsef Kadlecsik <kadlec@blackhole.kfki.hu>
Date: Sun, 13 Mar 2016 12:10:47 +0100
Subject: [PATCH] Fix "configure" breakage due to pkg-config related changes

The support for older pkg-config packages broke the "configure" script
(reported by Jan Engelhardt).
---
 configure.ac | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configure.ac b/configure.ac
index 1988836..b05646b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -12,6 +12,8 @@ LT_INIT([dlopen])
 LT_CONFIG_LTDL_DIR([libltdl])
 LTDL_INIT([nonrecursive])
 
+PKG_PROG_PKG_CONFIG
+
 dnl Shortcut: Linux supported alone
 case "$host" in
 *-*-linux* | *-*-uclinux*) ;;
-- 
2.7.2

